src/Compilers/CSharp/Portable/Parser/SyntaxParser.cs

```diff
@@ -1058,30 +1058,16 @@ protected TNode CheckFeatureAvailability<TNode>(TNode node, MessageID feature, b
             }
 
             var featureName = feature.Localize();
-            var requiredFeature = feature.RequiredFeature();
```

Comment:
Please do not remove this from code from the compiler. It is intended to facilitate experimental code residing in branches of the compiler, even though the product code does not contain experiments.